shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -157,6 +157,12 @@ - (instancetype)init {
 
 - (void)sharedSetupWithProject:(nullable FlutterDartProject*)project
                   initialRoute:(nullable NSString*)initialRoute {
+  // Need the project to get settings for the view. Initializing it here means
+  // the Engine class won't initialize it later.
+  if (!project) {
+    project = [[FlutterDartProject alloc] init];
```

Comment:
This object is leaked when passed to `[FlutterEngine initWithName:project:allowHeadlessExecution:]` as that ctor retains the `project` arg when not `nil`. Either `autorelease` the object or use a `scoped_nsobject` for the same. Engine TUs don't use ARC.

Reply:
Ahh good catch, fixed with autorelease.